x-pack/legacy/plugins/infra/public/containers/logs/log_analysis/log_analysis_jobs.tsx

```diff
@@ -43,14 +33,19 @@ export const useLogAnalysisJobs = ({
   spaceId: string;
   timeField: string;
 }) => {
-  const [jobStatus, setJobStatus] = useState<AllJobStatuses>(getInitialJobStatuses());
+  const [jobStatus, setJobStatus] = useState<Record<JobType, JobStatus>>({
+    'log-entry-rate': 'unknown',
+  });
   const [hasCompletedSetup, setHasCompletedSetup] = useState<boolean>(false);
 
   const [setupMlModuleRequest, setupMlModule] = useTrackedPromise(
     {
       cancelPreviousOn: 'resolution',
       createPromise: async (start, end) => {
-        setJobStatus(getInitialJobStatuses());
+        setJobStatus(currentJobStatus => ({
+          ...currentJobStatus,
+          'log-entry-rate': 'initializing',
```

Comment:
The only part I'm not sure about is this, and it's maybe not a big problem overall. We use `jobSummary.jobState === 'opening'` to determine a `jobStatus` state of `initializing`, my understanding is this would be when everything has been created successfully my ML, but is still opening up. But here we're setting that in the `createPromise` of `setupMlModule` when things haven't been created yet. Maybe `unknown` would be a better state at this point?

Reply:
If we look at the `JobStatus` as an indicator of the current stage of the (abstract) job/datafeed lifecycle, wouldn't `unknown` be incorrect? We know the state of the job, which is that it's being created but not running yet. Do you think we need to differentiate that more or that there is a risk in decoupling our abstract job state too much from the technical ML job state?

Reply:
> We know the state of the job, which is that it's being created but not running yet.

Okay, put like that it makes sense 👍 No, lets not add another state if it's not needed. Thanks for the explanation.